include/deal.II/fe/fe_values.h

```diff
@@ -292,6 +316,13 @@ namespace FEValuesViews
                               std::vector<typename ProductType<value_type,typename InputVector::value_type>::type> &values) const;
 
     /**
+     *
+     */
+    template <class InputVector>
+    void get_function_values_from_local_dof_values (const InputVector &fe_function,
```

Comment:
Or maybe, if you want to be extra clear, `local_dof_values` given that you typically get them out of something like `cell->get_local_dof_values()`.